src/pages/components/video-player.mdx

```diff
@@ -29,6 +30,15 @@ The Video player aspect ratio should match that of the video, or use one of the
 </Column>
 </Row>
 
+## Content guidance
```

Comment:
I struggle a little here with this particular component because it's all auto-populated from the Kaltura API, right? Do we display what's there, or what they can change, or nothing at all?

A couple things that maybe missing...

- [ ] Thumbnail (from the API)
- [ ] Thumbnail alt (from the API) [title and duration]

Reply:
Yeah I was thinking about adding that in the content model, but i was thinking that was all populated through Kaltura so it's not something the user necessarily has to worry about from a component perspective.
If we think it is, I can totally add it in.

Reply:
Ok, so by that logic I don't think I would include `caption` since that's coming from the API as well.

Reply:
Approved :)